tests/knowledge.test.ts

```diff
@@ -98,6 +98,46 @@ test('deletes knowledge', async ({ page }) => {
 	await expect(knowledgeSelect).toContainText(MOCK_KNOWLEDGE[1].name);
 });
 
+test('all knowledge can be deleted', async ({ page }) => {
+	await page.goto('/knowledge');
+		await expect(page.getByText('No knowledge')).toBeVisible();
+		await expect(page.getByTestId('knowledge-item')).toHaveCount(0);
+
+		// Stage 2 knowledge
+		await page.evaluate(() => window.localStorage.setItem(
+			'hollama-knowledge',
+			JSON.stringify(
+				[
+					{
+						id: "qbhc0q",
+						name: "Test",
+						content: "Hello World",
+						updatedAt: new Date().toISOString()
+					},
+					{
+						id: "m2jjac",
+						name: "Test 2",
+						content: "Hello World 2",
+						updatedAt: new Date().toISOString()
+					}
+				]
+			)
+		));
```

Comment:
Minor thing, import [`MOCK_KNOWLEDGE`](https://github.com/fmaclen/hollama/blob/main/tests/utils.ts#L135-L148) and use that to seed the local storage.

One thing to keep in mind is that `page.evaluate()` runs _"inside the browser"_ so you won't have `MOCK_KNOWLEDGE` available, instead you need to pass it as an argument. Here's [an example](https://github.com/fmaclen/hollama/blob/main/tests/session.test.ts#L159-L183).